docs/using/security-non-events.md

```diff
@@ -0,0 +1,58 @@
+
+Etherpad has had actual bugs that have been mitigated on Sandstorm:
+
+* [CVE-2015-3297](http://www.openwall.com/lists/oss-security/2015/04/12/1), [4085](http://www.openwall.com/lists/oss-security/2015/05/26/3): Path injection bugs could allow remotely dumping arbitrary files from the server's filesystem, including the Etherpad database, leaking pad contents and user credentials (session tokens). On Sandstorm, Etherpad can only see its own package contents and database, not files belonging to other apps or the system, so it can only leak its own data. The main damage from that -- leaking pad contents -- doesn't matter on Sandstorm because to launch the attack the attacker would need read access already (as described above). Meanwhile, leaking session tokens could normally (outside Sandstorm) allow impersonation of other users, but on Sandstorm Etehrpad relies on Sandstorm for authentication, and only uses session tokens to disambiguate anonymous users. Thus, on Sandstorm, this bug could be exploited to allow one user with edit access to impersonate some other anonymous user (forging edit history), but not to impersonate a logged-in user.
+
+* [CVE-2015-2298](http://www.openwall.com/lists/oss-security/2015/03/15/3): Due to a malformed database query, a request to export a pad by ID would in fact return contents of all pads on the server which had the specified ID as a substring of their own. Thus, all pads on the server could be dumped by specifying a series of one-character IDs (or maybe in a single query with an empty ID). This bug is irrelevant on Sandstorm because the attacker would need already to have read access to launch the attack (as described above).
+
+## Wordpress
+
+The Wordpress app on Sandstorm is used to publish public web sites. However, the app does not respond to individual page views of that site. Instead, the app generates static content which it hands off to Sandstorm, and Sandstorm serves the site statically. The site administrator accesses Wordpress's administrative interface through Sandstorm like any other app, but visitors to the public site visit a separate hostname where they don't see Sandstorm at all. The site owner may share access to the edit UI but would never share this access publicly.
```

Comment:
This paragraph is a little confusing but I can live with it. I can see if I can propose a rewording, and that doesn't have to block merging this.